src/server/pachyderm_test.go

```diff
@@ -3079,8 +3079,6 @@ func TestStopPipeline(t *testing.T) {
 }
 
 func TestStandby(t *testing.T) {
```

Comment:
It seems like `Standby` is now just `Autoscaling` - should probably rename the test as well.